nginx/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM jonasal/nginx-certbot:5.4
```

Comment:
> ℹ️ Please use a [specific tag](https://github.com/JonasAlfredsson/docker-nginx-certbot/blob/master/docs/dockerhub_tags.md) when doing a Docker pull, since :latest might not always be 100% stable.

from [docker-nginx-certbot readme](https://github.com/JonasAlfredsson/docker-nginx-certbot?tab=readme-ov-file#docker-nginx-certbot)